src/printer.rs

```diff
@@ -121,12 +127,20 @@ impl Printer {
                 // In terse mode, nothing is printed before the job. Only
                 // `print_single_outcome` prints one character.
             }
+            FormatSetting::Json => {
+                writeln!(
+                    self.out,
+                    "{{ \"type\": \"test\", \"event\": \"started\", \"name\": \"{}\" }}",
+                    name
```

Comment:
But what if the name contains a `"`? Or `\"`? The `name` needs to be properly escaped. [This crate](https://crates.io/crates/escape8259) looks like a perfect fit for this: it contains exactly the escaping logic we need without the full serde serialization logic. (I haven't looked too deeply into it yet though, so please check that)